File: project/leogreen_server/src/main/java/at/htl/model/pojo/Question.java
package at.htl.model.pojo;

import lombok.Data;

import java.io.Serializable;
import java.util.List;

@Data
public class Question implements Serializable {
    Long id;
    String title;
    List<Answer> answers;
    Answer correctAnswer;

    public Question(Long id, String title, List<Answer> answers, Answer correctAnswer) {
        this.id = id;
        this.title = title;
        this.answers = answers;
        this.correctAnswer = correctAnswer;
    }
}

File: project/client/leoGreen/src/app/model/question.ts
import {Answer} from "./answer";

export class Question{
  id: number
  title: string
  answers: Answer[]
  correctAnswer: Answer

  constructor(id: number, title: string, answers: Answer[], correctAnswer: Answer) {
    this.id = id;
    this.title = title;
    this.answers = answers;
    this.correctAnswer = correctAnswer;
  }
}
